CHANGELOG.md

```
@@ -2,6 +2,80 @@
+
+### Documentation
+
+- Update PeerStore::insert doc-comment to mention blocks by @cdunster
+
+### Automated Changes
+
+- *(deps)* Bump actions/checkout from 4 to 5 by @dependabot[bot] in [#299](https://github.com/holochain/kitsune2/pull/299)
+  - Bumps [actions/checkout](https://github.com/actions/checkout) from 4 to 5. - [Release notes](https://github.com/actions/checkout/releases) - [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md) - [Commits](https://github.com/actions/checkout/compare/v4...v5)
+  - Updated-dependencies: - dependency-name: actions/checkout   dependency-version: '5'   dependency-type: direct:production   update-type: version-update:semver-major ...
+
+# Changelog
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)